### ISSUE TITLE
AttributeError with Sphinx v4.2.0 and ablog 0.10.27

### ISSUE THREAD

**Issue by tomwhite** [sunpy/ablog#151]
### Describe the bug

Building with Sphinx v4.2.0 and ablog 0.10.27 results in an `AttributeError`.

### What happened?

```
Exception occurred:
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/ablog/post.py", line 160, in apply
    docinfo = list(self.document.findall(nodes.docinfo))
AttributeError: 'document' object has no attribute 'findall'
```

(More information at https://github.com/pystatgen/sgkit/runs/7616945453?check_suite_focus=true)

Looking at the history of that file the relevant change is here:

https://github.com/sunpy/ablog/commit/a3a8fb932dae00cd7766bd38554f98947104f37b#diff-f4d8728fd1bca178eb101f7635407e0b843bb4d18d2df7230352f1d5583e4f39

I haven't looked further, but it seems that there's some version mismatch. Rolling back to ablog 0.10.26 fixes the problem.

### Installation method

pip

**Comment by nabobalis** [comment]
Right, I made a horrible assumption on sphinx version and docutils.
I will patch this.

What version of docutils is being used in that build?
Would it be possible to update to sphinx 5?

**Comment by tomwhite** [comment]
Thanks @nabobalis.

This was with docutils 0.17.1. It might be possible to update sphinx, but it's not a priority at the moment as the doc build is quite brittle.

**Comment by nabobalis** [comment]
The quickest fix will be to pin the ablog version below 0.10.27, hopefully I patch this soon.

**Comment by nabobalis** [comment]
What I am worried about is why the CI did no catch this.

I will need to work out why.
